srcs/juloo.keyboard2/KeyValue.java

```diff
@@ -42,10 +42,11 @@
   public static final int FLAG_ACCENT_SUPERSCRIPT = (1 << 22);
   public static final int FLAG_ACCENT_SUBSCRIPT = (1 << 23);
   public static final int FLAG_ACCENT_RING = (1 << 24);
+  public static final int FLAG_ACCENT_SZLIG = (1 << 25);
```

Comment:
As you said, having an accent for this is annoying: need two gestures to type the `ß` character.

Adding that character on all the layouts would be fine, but we need a way to remove it for non-German users. (I'll work on that part)

Reply:
Thank you!